### ISSUE TITLE
HCP Pangeo on AWS

### ISSUE THREAD

**Issue by arokem** [pangeo-data/pangeo-cloud-federation#416]
Hello! Thanks to a recently-funded BRAIN grant (another one!) I am developing a new deployment of Pangeo that will target the [Human Connectome Project](https://registry.opendata.aws/hcp-openaccess/). Since the data is already on AWS, this deployment will also live on AWS (specifically, in the us-east-1 region). With help from @scottyhq (thanks!), I am already deploying the cluster on eks, with eksctl (well, trying... config is here: https://github.com/nrdg/hcp-pangeo).

Question: should I add my deployment's stuff here? If the answer is yes, could someone send along the git-crypt thingee? Thanks!

**Comment by jhamman** [comment]
Hi @arokem! I'm sorry this sat until now. 

*tldr; I think you might be better off maintaining a clone of this project with just your hub in it. *

I think its in scope to add additional hubs (like your HCP hub) here but let me it try to articulate some of the pros and cons.

**pros**
- joint maintenance of the underlying infrastructure helm/chart
- periodic updates using a pretty slick CICD system
- a bit of a community of maintainers to help when things go wrong

**cons**
- things break, usually not on your cluster, but this can slow down development
- there is actually not much shared infrastructure between the hubs so we could pretty easily split them out

Ideally, we could create a template for a single project (like HCP) to copy and get started. Perhaps down the road, you'll have more hubs in the project and you'll want to keep them in sync. This is what this project (hubploy/pangeo-cloud-federation) is going to be best for.

So, I think its up to you. I'd be keen to help you get a clone of this project setup. Hopefully we learn from your process. Thoughts?
